packages/paymaster/src/utils/Types.ts

```diff
@@ -103,11 +104,9 @@ export type FeeQuotesOrDataResponse = {
 
 export type PaymasterAndDataResponse = {
   paymasterAndData: string;
-  preVerificationGas?: BigNumberish;
-  verificationGasLimit?: BigNumberish;
-  callGasLimit?: BigNumberish;
-  maxPriorityFeePerGas?: BigNumberish;
-  maxFeePerGas?: BigNumberish;
+  preVerificationGas?: string;
```

Comment:
do we receive strings from our paymaster service?

Reply:
updated